channels.go

```diff
@@ -0,0 +1,56 @@
+package helix
+
+/*
+This file makes up for a missing API in Helix. This appears to be a recent
+change (as of around 2020-08-03) to the Twitch API, where GetStreams no longer
+returns information about offline channels.
+*/
```

Comment:
Can we remove this comment from the source code and instead place it in the description of the PR?

Reply:
Ah, yes of course. It was for my own notes, and I forgot to remove it.